Neos.ContentRepository.Export/src/Asset/AssetExporter.php

```diff
@@ -14,7 +16,8 @@ public function __construct(
         private readonly Filesystem $files,
         private readonly AssetLoaderInterface $assetLoader,
         private readonly ResourceLoaderInterface $resourceLoader,
-    ) {}
+    ) {
+    }
```

Comment:
I sneaked in some cosmetic fixes to allow us to enable PHPCS for this package, too